azkaban-web-server/src/web/js/azkaban/view/project-permissions.js

```diff
@@ -148,12 +148,10 @@ azkaban.ChangePermissionView = Backbone.View.extend({
     $('#change-permission-error-msg').hide();
   },
 
-  display: function (userid, newPerm, group, proxy) {
```

Comment:
I believe we can delete also lines: 29, 317, 322, 326, 327. Basically 'proxy' property is not used in PermissionTableView.
Also please remove parameter proxy from line 39.